src/core/core.scale.js

```diff
@@ -538,6 +542,40 @@ module.exports = Element.extend({
 		return rawValue;
 	},
 
+	_parseValue: function(raw) {
+		var value, start, end, min, max;
+
+		if (helpers.isArray(raw)) {
+			value = this.getRightValue(raw);
+			start = value[0];
```

Comment:
Based on [my previous comment](https://github.com/chartjs/Chart.js/pull/6056/files#r280851583):

```javascript
start = +value[0];
end = +value[1];
```

Reply:
Actually, do we need `value = this.getRightValue(raw);` instead of `value = raw`?

Then call:

```javascript
start = +me.getRightValue(value[0]);
end = +me.getRightValue(value[1]);
```

Reply:
done